### PR TITLE
[Woo POS] Changes to more screen to accomodate entry point to POS

### LINKED ISSUE #11545: [Woo POS] Add "new" red badge, change icon and change position for the POS button in the "more" screen
p91TBi-beb-p2

Image

### PR DESCRIPTION
**Pull request by kidinov**
Closes: #11545

### Description

The PR adds changes to implement an updated UI for the "more" menu screen and adds POS button to it.

As requested by Joe also added "external link" icons for Wp-admin and view store.

The red badge is not implemented here.

image

### Testing instructions

* Go to the More screen and make sure there is no changes apart from the UI. Check dark mode. Make sure that all the buttons are in place as they used to be except the POS is on top now

### Images/gif

https://github.com/woocommerce/woocommerce-android/assets/4923871/cc39ac85-15f7-4a23-9d59-112e872f67e4

image

- [x] I have considered if this change warrants release notes and have added them to `RELEASE-NOTES.txt` if necessary. Use the "[Internal]" label for non-user-facing changes.

### CODE REVIEW

**Comment by dangermattic** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">This PR is larger than 300 lines of changes. Please consider splitting it into smaller PRs for easier and faster reviews.</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_pr-check">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

File: WooCommerce/src/main/kotlin/com/woocommerce/android/ui/moremenu/MoreMenuScreen.kt [hunk 1]
```diff
@@ -416,65 +421,90 @@ private fun createBadgeEnterAnimation(): EnterTransition {
...
-                R.drawable.ic_more_menu_reviews,
-                BadgeState(
-                    badgeSize = R.dimen.major_150,
-                    backgroundColor = R.color.color_primary,
-                    textColor = R.color.color_on_primary,
-                    textState = TextState("3", R.dimen.text_minor_80),
-                    animateAppearance = false
+    WooThemeWithBackground {
+        val state = MoreMenuViewState(
+            menuSections = listOf(
```
Comment (backwardstruck) [inline WooCommerce/src/main/kotlin/com/woocommerce/android/ui/moremenu/MoreMenuScreen.kt:413]: I think `MoreMenuViewModelTests` needs to be updated now that `generalMenuItems` is now `menuSections`:

```
[2024-05-22T09:45:49Z] e: file:///var/lib/buildkite-agent/builds/ci-cluster-android-i-0b1d9a8e3ea86d90c-1/automattic/woocommerce-android/WooCommerce/src/test/kotlin/com/woocommerce/android/ui/moremenu/MoreMenuViewModelTests.kt:119:44 Unresolved reference: generalMenuItems
 ```

<br class="Apple-interchange-newline">

**Review by backwardstruck — COMMENTED** [review]
For some reason I'm not seeing the latest code changes, it seems. Maybe you need to push?

Screenshot_20240522_161926

**Comment by kidinov** [comment]
@backwardstruck  could please take a look again - I've made small changes to look it an Android design (before I actually changed it to an iOS design without realizing it)

File: WooCommerce/src/main/kotlin/com/woocommerce/android/ui/moremenu/MoreMenuScreen.kt (hunk 1 above)
Comment (kidinov) [reply WooCommerce/src/main/kotlin/com/woocommerce/android/ui/moremenu/MoreMenuScreen.kt:413]: Yep, changed that

**Review by backwardstruck — APPROVED** [review]
Tested and code changes look good.
Screenshot_20240527_105355

**Pull request merged**